### ISSUE TITLE
Print fixture to hold motor during OneShot testing.

### ISSUE THREAD

**Issue by ebelski** [ebelski/rust-copter#15]
The torque from the motor causes it to jump around on the desktop when spinning so it needs fixed in place to prevent damage to the electronics.

**Comment by mciantyre** [comment]
Will we be versioning 3D models in the repo? If so, we can set up [Git Large File Storage](https://git-lfs.github.com) to help manage the size of the repo.

**Comment by ebelski** [comment]
It's probably best to version our solid models. The fixture just finished printing so once I verify the fit, I'll upload the stl and step files.
![motor fixture](https://user-images.githubusercontent.com/19614064/84326022-123ae280-ab4a-11ea-97b5-6ebbfacd84aa.jpg)

**Comment by ebelski** [comment]
It works! We probably only need to secure the motor with 2 bolts, but I've included all four bolt holes just as an added factor of safety. I printed it with 20% infill but realistically it could probably make due with only 10%. The part is 20 mm tall, but could be shorter. The jaws on my vice are 20 mm tall so I made it congruent with them. Overall I'm pleased with the fit and function.

![fixture in vice](https://user-images.githubusercontent.com/19614064/84326746-b3766880-ab4b-11ea-947b-51b281cd487e.jpg)
